Services/JwtUserService.cs

```diff
@@ -104,6 +104,13 @@ public async Task<JwtUserDto> ProcessUserAsync(User user)
             };
         }
 
+        public async Task<bool> DoesNameOrEmailExist(string nameOrEmail)
+        {
+            IEnumerable<User> user = await _unitOfWork.UserRepository.ReadByConditionAsync(user => user.Name == nameOrEmail || user.Email == nameOrEmail);
+
+            return user.Count() > 0;
```

Comment:
You can use user.Any(). Also I think it would be better to use "users" because of IEnumerable.

Reply:
Changed to `users.Any()`.